doc/classes/GraphFrame.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="GraphFrame" inherits="GraphElement" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		GraphFrame is a special [GraphElement] that can be used to organize other [GraphElement]s inside a [GraphEdit].
+	</brief_description>
+	<description>
+		GraphFrame is a special [GraphElement] to which other [GraphElement]s can be attached. It can be configured to automatically resize to enclose all attached [GraphElement]s. If the frame is moved, all the attached [GraphElement]s inside it will be moved as well.
+		A GraphFrame is always kept behind the connection layer and other [GraphElement]s inside a [GraphEdit].
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="get_titlebar_hbox">
+			<return type="HBoxContainer" />
+			<description>
+				Returns the [HBoxContainer] used for the title bar, only containing a [Label] for displaying the title by default.
+				This can be used to add custom controls to the title bar such as option or close buttons.
+			</description>
+		</method>
+	</methods>
+	<members>
+		<member name="autoshrink_enabled" type="bool" setter="set_autoshrink_enabled" getter="is_autoshrink_enabled" default="true">
+			If [code]true[/code], the frame's rect will be adjusted automatically to enclose all attached [GraphElement]s.
+		</member>
+		<member name="autoshrink_margin" type="Vector2i" setter="set_autoshrink_margin" getter="get_autoshrink_margin" default="Vector2i(40, 40)">
+			The margin around the frame (horizontal, vertical) that is used to calculate the size of the frame when [member autoshrink_enabled] is [code]true[/code].
+		</member>
+		<member name="drag_margin" type="int" setter="set_drag_margin" getter="get_drag_margin" default="16">
+			The margin around the frame that can be used to drag the frame.
+		</member>
+		<member name="mouse_filter" type="int" setter="set_mouse_filter" getter="get_mouse_filter" overrides="Control" enum="Control.MouseFilter" default="0" />
+		<member name="tint_color" type="Color" setter="set_tint_color" getter="get_tint_color" default="Color(0.3, 0.3, 0.3, 0.75)">
+			The color of the frame when [member tint_color_enabled] is [code]true[/code].
+		</member>
+		<member name="tint_color_enabled" type="bool" setter="set_tint_color_enabled" getter="is_tint_color_enabled" default="false">
+			If [code]true[/code], the tint color will be used to tint the frame.
+		</member>
+		<member name="title" type="String" setter="set_title" getter="get_title" default="&quot;&quot;">
+			Title of the frame.
+		</member>
+		<member name="title_centered" type="bool" setter="set_title_centered" getter="is_title_centered" default="true">
+			If [code]true[/code] the title is centered. Otherwise, the title is aligned to the left or right.
+		</member>
```

Comment:
In https://github.com/godotengine/godot/pull/88014#discussion_r1481433307 I did agree with you and said that this property should probably not exist. If users need it they will ask for it.

Reply:
Removed it.